### DISCUSSION TITLE
Allow the CLI to generate polymorphic relations

### DISCUSSION THREAD

**Discussion by achrinza** [loopbackio/loopback-next#8463] [Ideas]
This is a continuation of https://github.com/loopbackio/loopback-next/issues/2487.

At this moment, it's not possible for the LoopBack 4 CLI to generate _polymorphic relations_. We should update the CLI to introduce a new set of prompts for the _relation generator_ to ask if the user wants to make the relation polymorphic, and if so, what discriminator should be used.

**Comment by j314h** [comment]
hello, is this still in progress, or has this been implemented?

**Comment by OnTheThirdDay** [comment]
This is still in progress. BTW, the polymorphic relations feature has a bug that I am working on a fix.

**Reply by j314h** [reply]
OK, thanks. thank you for your work. I think there is a problem in the documentation.
I'm testing, I'll see if there is a problem.

**Comment by OnTheThirdDay** [comment]
In case anything unclear in the doc, you can have a check of this example. https://github.com/OnTheThirdDay/lb4-polymorphic-example
